product_docs/docs/biganimal/release/reference/cli/using_features.mdx

```diff
@@ -70,6 +76,12 @@ __OUTPUT__
 ? Volume properties: P1 (4 Gi, 120 Provisioned IOPS, 25 Provisioned MB/s)
 ? Networking: Public
 ? By default your cluster allows all inbound communications, add IP allowed list to restrict the access: No
+? Cloud Provider Subscription ID: 
+- 123123123
+- 456456456
+? Service Account IDs, (leave empty to stop adding):
+- service-account-1234567b@development-data-123456.iam.gserviceaccount.com
+- service-account-1234567b@development-data-123456.iam.gserviceaccount.com
```

Comment:
[same as above](https://github.com/EnterpriseDB/docs/pull/4812/files#r1378600370)

Reply:
Should be good now.